openhtf/io/frontend/src/app/station.header.ts

```diff
@@ -0,0 +1,34 @@
+// Copyright 2016 Google Inc.All Rights Reserved.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+
+//     http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+import {Component, Input} from 'angular2/core';
+import {NumberPipe}       from 'angular2/common';
+
+import {Countdown} from './utils';
+
+import 'file?name=/templates/station.header.html!./station.header.html';
```

Comment:
The demo code had many style inconsistencies. I just picked one of the conventions arbitrarily and randomly extended it as the situation required. I'll see if I can dig up a ts style guide.